test/sparse/sparse.jl

```diff
@@ -35,8 +35,8 @@ do33 = ones(3)
         sqrfloatmat, colfloatmat = sprand(4, 4, 0.5), sprand(4, 1, 0.5)
         @test_throws DimensionMismatch (+)(sqrfloatmat, colfloatmat)
         @test_throws DimensionMismatch (-)(sqrfloatmat, colfloatmat)
-        @test_throws DimensionMismatch min(sqrfloatmat, colfloatmat)
-        @test_throws DimensionMismatch max(sqrfloatmat, colfloatmat)
+        # @test_throws DimensionMismatch min(sqrfloatmat, colfloatmat) # vectorized min no longer exists
+        # @test_throws DimensionMismatch max(sqrfloatmat, colfloatmat) # vectorized max no longer exists
```

Comment:
Sure, though those tests would be redundant with the similar tests #19690 introduces. Preference for redundancy or removal? Thanks!